connection.js

```diff
@@ -371,7 +373,7 @@ Connection.prototype._process_data = function() {
                 this.logdebug('[early_talker] state=' + this.state + ' esmtp=' + this.esmtp + ' line="' + this_line + '"');
             }
             this.early_talker = true;
-            self._process_data();
+            nextTick(self._process_data);
```

Comment:
No it must. Otherwise it's a function call not a method call.

> On Sep 28, 2015, at 7:29 AM, Josef Fröhle <notifications@github.com> wrote:
> 
> In connection.js:
> 
> > @@ -371,7 +373,7 @@ Connection.prototype._process_data = function() {
> >                  this.logdebug('[early_talker] state=' + this.state + ' esmtp=' + this.esmtp + ' line="' + this_line + '"');
> >              }
> >              this.early_talker = true;
> > -            self._process_data();
> > +            nextTick(self._process_data);
> It can be done so, but must not. So save some CPU time.
> 
> —
> Reply to this email directly or view it on GitHub.
>